imap_data_access/processing_input.py

```diff
@@ -376,11 +378,50 @@ def get_science_files(self) -> list[ProcessingInput]:
         Returns
         -------
         out : list[ProcessingInput]
-            list of ScienceInput files contained in the collection.
+            List of ScienceInput files contained in the collection.
         """
         out = []
         for file in self.processing_input:
             if file.input_type == ProcessingInputType.SCIENCE_FILE:
                 out.append(file)
+        return out
+
+    def get_files(
```

Comment:
Maybe call the above `get_science_inputs()` as suggested by @lacoak21 
```suggestion
    def get_file_paths(
```